themes/10up-block-theme/phpcs.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<ruleset name="10up PHPCS">
+	<description>10up PHPCS extended.</description>
+	<exclude-pattern>dist/</exclude-pattern>
+	<exclude-pattern>plugins/*</exclude-pattern>
```

Comment:
Why is this here? 

Also do we even need this file if there is one at the root of the project?

Reply:
This looks incorrect, based on the fact that it's excluding `plugins/*`, _but_, I do think there's some benefit to this, especially if we move ahead with something like https://github.com/10up/10up-toolkit/issues/424, as it would mean any themes/plugins pulled in via that command would have valid PHPCS configs, even if pulled into legacy codebases.